packages/react/src/modules/in-app-messages/components/Banner/Banner.tsx

```diff
@@ -70,9 +70,11 @@ const Body: React.FC<{ body: string } & React.ComponentPropsWithRef<"div">> = ({
   ...props
 }) => {
   return (
-    <div className={clsx("knock-iam-banner__body", className)} {...props}>
-      {body}
-    </div>
+    <div
+      className={clsx("knock-iam-banner__body", className)}
+      dangerouslySetInnerHTML={{ __html: body }}
```

Comment:
Agreed this works well for the OOTB schemas that will now be markdown. Are there any drawbacks though in terms of being able to reuse these components for custom schemas? Wondering if it'd be worth it to provide an escape mechanism to still render e.g. `body` as a regular child component (or keep that as the regular and only render using inner html for the 'default' component that is tied to the OOTB schemas). I'd lean towards not worrying about it at the moment.

Reply:
> Are there any drawbacks though in terms of being able to reuse these components for custom schemas?

My thinking was that you would use these components if you are using the ootb message type with the provided schemas provided as is. You can customize rendering side of the ootb message types in various ways, including using your own component(s). But if you have a custom schema then you should bring your own component that matches the schema.